.github/workflows/go.yml

```
@@ -61,140 +61,140 @@ jobs:
+        needs: test
+        runs-on: ubuntu-latest
+        name: Check Coverage
+        steps:
+            - name: checkout
+              uses: actions/checkout@v4
+
+            - name: Download cover profile artifact
+              id: download-coverage
+              uses: actions/download-artifact@v4
+              with:
+                name: coverage.out
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)